5740-2020-Robot/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -33,10 +32,10 @@
  */
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
-
-  private final Drivetrain m_drivetrain = new Drivetrain(); // Robot Drivetrain
-  private final DriveSlowly m_autoCommand = new DriveSlowly(m_drivetrain);
-  //private final ExampleCommand m_autoCommand = new ExampleCommand(m_exampleSubsystem);
+  private ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
```

Comment:
What is up with using this example subsystem? create or rename this to something that we will use in the actual code.